### ISSUE TITLE
[bug] app contains code not allowed by OwnCloud

### ISSUE THREAD

**Issue by rmbleeker** [owncloud/twofactor_totp#11]
**OwnCloud version:** 9.1.5.2
**OS version:** CentOS 7
**PHP version:** 7.0.10
**Two Factor TOTP version:** 0.4.2

When trying to install the latest *Two Factor TOTP Provider* plugin from within OwnCloud by going to **Apps > Tool** I get the following error:

```
{"reqId":"WO@DwebMDQyQFH1cnxehWQAAAAQ","remoteAddr":"xx.xx.xx.xx","app":"core","message":"App can't be installed because of not allowed code in the App","level":3,"time":"2017-04-13T15:57:53+02:00","method":"POST","url":"\/store\/index.php\/settings\/ajax\/enableapp.php","user":"xxxxxxxxxx"}
```

I can install the app after setting `'appcodechecker' => false,` in the *config.php*, but I'd rather have the app code checking feature active for security reasons.

**Comment by rmbleeker** [comment]
I *think* this is fixed with TwoFactor TOTP 0.4.3 on OwnCloud 10, but I don't dare uninstall and re-install the app to test it. If someone can confirm this issue could be closed.

**Comment by karakayasemi** [comment]
I tested that TOTP 0.4.3 can be activate when appcochecker is active. It works.
